internal/pkg/cli/flag.go

```diff
@@ -47,6 +47,16 @@ const (
 	svcPortFlag           = "port"
 
 	storageTypeFlag = "storage-type"
+
+	numFlag              = "num"
+	cpuFlag              = "cpu"
+	memoryFlag           = "memory"
+	imageFlag            = "image"
+	taskRoleFlag         = "task-role"
+	subnetIDFlag         = "subnet-id"
+	securityGroupIDsFlag = "security-group-id"
```

Comment:
What do you think about having these just be `subnet` and `security-groups`? Also, should we make the securityGroupID flag name consistently plural or singular?